src/main/java/core/basesyntax/db/FruitStorage.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import java.util.Map;
+import java.util.TreeMap;
+
+public class FruitStorage {
+    public static final Map<String, Integer> FRUIT_STORAGE = new TreeMap<>();

```

Comment:
Interesting decision to use TreeMap here

Reply:
Oh, I just wanted my report be automatically sorted in alphabetical order